images/onnx-serve-gpu/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM nvidia/cuda:9.1-cudnn7-devel
+
+ARG ONNXRUNTIME_VERSION="0.4.0"
```

Comment:
Is there a reason you wanted to have `ONNXRUNTIME_VERSION` independently defined in the two Dockerfiles rather than pinning the version in `requirements.txt`?

Reply:
Yeah, for GPU we need to install onnxruntime-gpu and for cpu it is just onnxruntime. Rather than having two different requirements.txt, with the one package change, I decided to move it to Dockerfile. This pattern is also followed in our TF images where the dockerfile is responsible for dictating the version.

Reply:
Sounds good 👍